src/level.rs

```diff
@@ -0,0 +1,258 @@
+// Copyright 2014 The html5ever Project Developers. See the
```

Comment:
This file can have "Copyright 2017."  Also I just noticed that this says "The html5ever Project Developers" which I must have copy-pasted accidentally a long time ago.  Please change it to "The Servo Project Developers" here and elsewhere.  Thanks!